procedures/foreman/release.md.erb

```diff
@@ -44,15 +44,22 @@
   * [ ] Tag the release using [tag.sh](https://gist.github.com/tbrisker/d51cedea51ccbcca00e72c1fc61550fe) `tag.sh <%= full_version %>`
   * [ ] Push: `git push upstream <%= short_version %>-stable --follow-tags`
 * [ ] Run the Jenkins [Tarballs Release](https://ci.theforeman.org/job/tarballs-release/) to create tarballs
-* [ ] [Download](https://github.com/theforeman/theforeman-rel-eng/blob/master/download_tarballs), [inspect](https://github.com/theforeman/theforeman-rel-eng/blob/master/inspect_tarballs), [sign and upload detached signatures](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_tarballs)
+* [ ] Update release version similar to [here](https://github.com/theforeman/theforeman-rel-eng/commit/2029a9688da00d9c385c3438dd71b594ba5f728e)
+* [ ] Sign Tarballs
+  * [ ] [Download](https://github.com/theforeman/theforeman-rel-eng/blob/master/download_tarballs)
+  * [ ] [Inspect](https://github.com/theforeman/theforeman-rel-eng/blob/master/inspect_tarballs)
+  * [ ] [Sign and upload detached signatures](https://github.com/theforeman/theforeman-rel-eng/blob/master/sign_tarballs)
 
 Note: If for some reason there was an issue with the tarballs that required uploading new tarballs, CDN cache should be invalidated so that the builders use the updated tarballs.
 
 # Packaging a release
 
 [Background documentation](https://projects.theforeman.org/projects/foreman/wiki/Release_Process#Packaging-a-release)
-* [ ] Update [foreman-packaging](https://github.com/theforeman/foreman-packaging) [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging) and [deb/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_deb_packaging) branches
-* [ ] Trigger [release_packages in Jenkins](https://ci.theforeman.org/job/release_packages/) by calling [release_packages](https://github.com/theforeman/theforeman-rel-eng/blob/master/release_packages)
+
+* [ ] Update [foreman-packaging](https://github.com/theforeman/foreman-packaging)
+  * [ ] [rpm/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_rpm_packaging)
+  * [ ] [deb/<%= short_version %>](https://github.com/theforeman/theforeman-rel-eng/blob/master/bump_deb_packaging) branches
+* [ ] Trigger [release_packages in Jenkins](https://ci.theforeman.org/job/release_packages/) by calling [release_packages script](https://github.com/theforeman/theforeman-rel-eng/blob/master/release_packages). NOTE: This step will fail since packages need to be signed. Continue on to next steps
```

Comment:
Why would release_packages fail because of signing? It does not require anything signed.

It *does* fail because of `foreman-release` in client repos, but that's a bug, as it worked just fine before.

Reply:
fwiw, I just tested, my laptop does not scratch build into client-el7, jenkins does. so we should investigate that.

Reply:
fixed by https://github.com/theforeman/foreman-packaging/pull/4481 https://github.com/theforeman/foreman-packaging/pull/4482 https://github.com/theforeman/foreman-packaging/pull/4483

Reply:
which now have been merged, so the addendum is not needed?